pom.xml

```diff
@@ -51,6 +51,8 @@
 ]]>
     </checkstyle.header.template>
     <findbugs.failOnViolation>true</findbugs.failOnViolation>
+    <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
+    <checkstyle.logViolationsToConsole>true</checkstyle.logViolationsToConsole>
```

Comment:
I couldn't find _checkstyle.logViolationsToConsole_ in here:
https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html

On this page, it is said the following property sets the value of _logViolationsToConsole_:
_checkstyle.console_

Am I referencing the wrong documentation?